docs/common-coreboot-docs/checkbox.md

```diff
@@ -0,0 +1,222 @@
+# Checkbox usage (Ubuntu 22.04)
+
+## Introduction
+
+Checkbox is a system testing platform for Ubuntu. It aims to provide a common
+framework to run all types of tests, from hardware tests, to command line tests,
+unit tests or desktop tests. This documentation describes usage of checkbox on
+Ubuntu 22.04, but it can work on older versions Ubuntu.
+
+## Download and install Checkbox
+
+1. Open terminal and type in below-described commands to install Checkbox and
+all its dependencies:
+
+    ```bash
+    $ sudo add-apt-repository ppa:hardware-certification/public
+    $ sudo apt-get update
+    $ sudo apt-get install checkbox-ng plainbox-provider-resource-generic \
+    plainbox-provider-certification-client plainbox-provider-checkbox \
+    canonical-certification-client
+    ```
+
+## Set up testing environment
+
+To achieve the best results from the tests and do many of them, we should fill
+as many internal and external slots as possible. Before running the checkbox,
+try to fill the slots if the DUT has one:
+
+External:
+* Memory Card Reader
+* USB Ports
+* Monitor Ports
+* Ethernet Ports
+* Headphone and Microphone Jacks
+* Thunderbolt Ports
+* Power Socket
+* PCMCIA or ExpressCard slot
+
+Internal:
+* DIMM/RAM slots
+* SATA ports
+* PCI/PCIe slots
+* M.2 slots
+* TPM header
+* Other slots
+
+In order for the tests to not require any interference after their startup,
```

Comment:
I think, that we should split this section into two separate: one with prerequisites (modules which could be necessary to do the tests) and the second with a description of how to prepare the environment.

Reply:
separated